### ISSUE TITLE
Not sorting in ruby erb for "class: 'hidden w-full'"

### ISSUE THREAD

**Issue by studious** [laytan/tailwind-sorter.nvim#109]
I have sorting work for most of my ruby eruby (.erb) files but the following class usage on the same page does not sort.

``` 
<%= link_to 'accounts', accounts_path, class: "pr-1 inline-block w-6 fill-indigo-600 hover:fill-indigo-900" %>
```

Should it sort and if I need to customise my nvim configs how do I go about doing this?

Thanks

**Comment by studious** [comment]
Ah, more digging and I think I see what's up. Embedded ruby in tree-sitter has no decent query mechanism. That's unfortunate. (output_directive (code) @cd)) is as close as I can get without more understanding.

Is that enough to potentially look for strings to sort?

**Comment by jbigler** [comment]
Yeah, the querying within the injected code is difficult. I used this and it seems to be working well enough for now:

`queries/eruby/tailwind.scm`
```
((output_directive
   (code) @ruby_code
   (#match? @ruby_code "class:"))
 (#gsub! @ruby_code ".*class:\\s*['\"]([^'\"]+)['\"].*" "%1")) @tailwind
```
